assets/index.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-use-before-define */
+function Book(title, author, pages, read = 'Unread') {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+let library = [];
+
+// Storage
+
+const populateStorage = () => {
+  library.push(new Book('Twist Of The Wrist', 'Keith Code', 135, 'Read'));
+  library.push(new Book('A Perfect Day', 'Lady Teresa', 140, 'Unread'));
+  localStorage.setItem('books', JSON.stringify(library));
+};
+// If storage is empty we store the demo books
+if (!localStorage.length) {
+  populateStorage();
+}
+
+library = JSON.parse(localStorage.getItem('books'));
+
+const view = {
```

Comment:
Let keep the application logic separated from DOM manipulation tasks.
You move the DOM tasks in another file.
![image](https://user-images.githubusercontent.com/31897434/75403616-812c5c80-5908-11ea-9145-78e898ebc929.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/